packages/ometrics/ometrics.0.1.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+license: "MIT"
+maintainer: "Valentin Chaboche <valentin.chaboche@lambda-coins.com>"
+homepage: "https://gitlab.com/nomadic-labs/ometrics"
+dev-repo: "git+https://gitlab.com/nomadic-labs/ometrics.git"
+bug-reports: "https://gitlab.com/nomadic-labs/ometrics/-/issues"
+synopsis: "OCaml analysis in a merge request changes"
+
+depends: [
+  "ocaml" {>= "4.12" & < "4.13"}
+  "dune" {>= "2.9.1"}
+  "yojson" {>= "1.7.0"}
+  "menhirSdk" {>= "20211012" & < "20211013"}
+  "menhirLib" {>= "20211012" & < "20211013"}
+  "menhir" {>= "20211012" & < "20211013"}
```

Comment:
Are you sure you need so strict bounds here?

Reply:
Oh, I see. If you are vendoring merlin, you should instead add a conflict:
```
conflict: [
  "menhir"
]
```

In general (as far as I know) there can be issues/complex conflicts when you have a dependency both installed and vendored.

If you think only a specific set of versions have issues, you can do
```
conflict: [
  "menhir" {!= "20211012"}
]
```

Reply:
I first (locally) tested with
```
conflicts: [
  "menhir"
]
```
but I got the same error about versions. It worked fine with your second suggestion.